src/translations/ru/happyWizardForm.js

```diff
@@ -0,0 +1,44 @@
+export default {
+  'happyWizardForm.btnNext': 'Вперед',
```

Comment:
I think these are general things, so move them to something like `/src/translations/en/app/`

`'common.button.next': 'Next',`